tangelo/linq/translator/translate_circuit.py

```diff
@@ -36,6 +36,8 @@
     "qiskit": translate_c_to_qiskit,
     "qulacs": translate_c_to_qulacs,
     "pennylane": translate_c_to_pennylane,
+    "stim": translate_c_to_stim,
+    "pennylane": translate_c_to_pennylane,
```

Comment:
you duplicated the Pennylane line. Maybe tableau or something was supposed to go there